src/html-conventions-lib/common-elements-html-conventions.xsl

```diff
@@ -369,21 +370,71 @@
                 "/>
 
     </xsl:template>
-    
+
     <xd:doc>
-        <xd:desc>[common-visibility-18] - The element $name$ is non-public. All elements shall be public.</xd:desc>
+        <xd:desc>[common-visibility-18] - The element $name$ is non-public. All elements shall be
+            public.</xd:desc>
         <xd:param name="element"/>
     </xd:doc>
     <xsl:template name="nonPublicElement">
         <xsl:param name="element"/>
         <xsl:variable name="elementScope" select="$element/@scope"/>
         <xsl:sequence
             select="
-            if ($elementScope = 'public') then
-            ()
-            else
-            f:generateHtmlWarning(fn:concat('The element ', $element/@name, ' is non-public. All elements shall be public '))"
+                if ($elementScope = 'public') then
+                    ()
+                else
+                    f:generateHtmlWarning(fn:concat('The element ', $element/@name, ' is non-public. All elements shall be public '))"
         />
     </xsl:template>
 
+    <xd:doc>
+        <xd:desc>[common-name-unique] - The name $value$ is not unique. The Concept names should be
```

Comment:
connectors are not in this file (2 logic's - attribute is still an element in the grant scheme of things. It is inheriting the same common checkers). The file name is suggesting what is inside (common element checkers). I think this should remain as it is because this is a common checker applied to elements (not including connectors).